paper/paper.md

```diff
@@ -0,0 +1,152 @@
+
+The object [`Semidiscretization`](https://trixi-framework.github.io/TrixiParticles.jl/stable/general/semidiscretization/) couples the passed systems to one simulation and also creates the corresponding neighborhood searches for each system. The resulting ODE problem is then fed into the time integrator and is solved by an appropriate numerical time integration scheme.
+
+
+![TODO: caption *Inspired by [docs Trixi](https://trixi-framework.github.io/Trixi.jl/stable/overview/#overview-semidiscretizations)* \label{fig:structure}](structure.png){width=100%}
+
+
+At present, TrixiParticles.jl includes the implementation of the following systems
+
+* *Fluid Systems*
+    + [`WeaklyCmpressibleSPHSystem` (WCSPH)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/weakly_compressible_sph/): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+    + [`EntropicallyDampedSPHSystem` (EDAC)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/entropically_damped_sph/): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
+
+* *Structure Systems*
+    + [`TotalLagrangianSPHSystem` (TLSPH)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/total_lagrangian_sph/): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
+    + `DEMSystem`: Discrete element system [@Bicanic:2004], [@Cundall:1979].
+
+* *Boundary Systems*
+    + [`BoundarySPHSystem`](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/boundary/) with several boundary models where each model follows a different interaction rule.
+    + `OpenBoundarySPHSystem`: System to simulate non-reflecting boundary conditions [@Lastiwka:2009]
+
+The code presented here can be found on [GitHub](https://github.com/trixi-framework/TrixiParticles.jl),  along with a detailed [manual](https://trixi-framework.github.io/TrixiParticles.jl/stable/) explaining how to use the package. Additionally, we provide tutorials explaining how to setup a simulation for fluids, structure mechanics or a fluid-structure interaction. All implemented examples for testing are located in the [`examples/`](https://github.com/trixi-framework/TrixiParticles.jl/tree/main/examples) folder, thus, the results can be easily reproduced.
+
+### Feature highlights
+
+As a young project that aims to be the Julia code for particle-based simulation we expect to integrate other methods such as incompressible SPH (ISPH) [@Liu:2022] or heat conduction with SPH [@Biriukov:2018] and couple them among each other and even with mesh-based methods.
+
+TODO: Instead of listing it in the paper, it might be better to link the [feature list](https://trixi-framework.github.io/TrixiParticles.jl/stable/#Features)
```

Comment:
That is not a good idea too only link there since the URL might change in the future.